jaxrs-clients/src/main/java/com/palantir/remoting3/jaxrs/AbstractFeignJaxRsClientBuilder.java

```diff
@@ -130,13 +131,14 @@ private Contract createContract() {
     }
 
     private static Decoder createDecoder(ObjectMapper objectMapper, ObjectMapper cborObjectMapper) {
-        return new Java8OptionalAwareDecoder(
-                new GuavaOptionalAwareDecoder(
-                        new InputStreamDelegateDecoder(
-                                new TextDelegateDecoder(
-                                        new CborDelegateDecoder(
-                                                cborObjectMapper,
-                                                new RejectNullDecoder(
-                                                        new JacksonDecoder(objectMapper)))))));
+        return new RejectNullDecoder(
```

Comment:
Thoughts on renaming this to "NeverReturnNullDecoder" ?

Reply:
sure, don't feel strongly either way